terraform/backend/module/alb.tf

```diff
@@ -1,12 +1,41 @@
+data "aws_region" "current" {}
+data "aws_caller_identity" "current" {}
+
+resource "aws_s3_bucket" "log-bucket" {
+  bucket_prefix = "${var.env}-${var.app}-external-alb-logs"
+}
+
+data "template_file" "log-bucket-policy" {
+  template = "${file("${path.module}/log-bucket-policy.json.tpl")}"
+
+  vars = {
+    alb_account_id = "${lookup(var.aws_region_map, "${data.aws_region.current.name}")}"
+    s3_arn         = "${aws_s3_bucket.log-bucket.arn}"
+    account_id     = "${data.aws_caller_identity.current.account_id}"
+  }
+
+resource "aws_s3_bucket_policy" "attach-policy-to-log-bucket" {
+  bucket = "${aws_s3_bucket.log-bucket.id}"
+  policy = "${data.template_file.log-bucket-policy.rendered}"
```

Comment:
would be better to put this policy in HCL https://github.com/codersagainstcovidorg/infra/blob/master/terraform/frontend/module/iam.tf#L65

Reply:
You want to add it as in-line bucket policy rather than templatizing as a separated document?